articles/spring-boot-smtp/index.md

```diff
@@ -0,0 +1,260 @@
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.4.3</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.queenter</groupId>
+    <artifactId>sendmail</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>sendmail</name>
+    <description>Demo project for Spring Boot</description>
+    <properties>
+        <java.version>11</java.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-mail</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-devtools</artifactId>
+            <scope>runtime</scope>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
+
+```
+
+### Mail configurations
+Now that we have set up the project, add the following configurations into the `applications. properties` file in the resources directory.
+
+```properties
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+spring.mail.username=email-address
+spring.mail.password=email-password
+# Other smtp properties
+spring.mail.properties.mail.smtp.auth=true
+spring.mail.properties.mail.smtp.connectiontimeout=5000
+spring.mail.properties.mail.smtp.timeout=5000
+spring.mail.properties.mail.smtp.writetimeout=5000
+# TLS port 587
+spring.mail.properties.mail.smtp.starttls.enable=true
+```
+
+Replace the `email-address and `email password` with your actual email address and password.
+
+For the above configurations to work in our applications, we must set up Gmail to allow connections from less secure apps.
+- On your Gmail account, click on `manage account`.
+- On the `manage account` screen click on the `security` menu item and select `turn on access to less secure apps` as shown below.
+
+![Turning on access to less secure apps](google-auth.png)
```

Comment:
Consider changing "![Turning on access to less secure apps](google-auth.png)" to "![Turning on access to less secure apps](/engineering-education/spring-boot-smtp/google-auth.png)"